tests/key_clef_tests.ts

```diff
@@ -6,7 +6,10 @@
 
 import { MAJOR_KEYS, MINOR_KEYS, TestOptions, VexFlowTests } from './vexflow_test_helpers';
 
-import { ContextBuilder, Glyph, KeySignature, Stave } from '../src';
+import { Glyph } from '../src/glyph';
+import { KeySignature } from '../src/keysignature';
+import { ContextBuilder } from '../src/renderer';
+import { Stave } from '../src/stave';
```

Comment:
Agree perhaps you colud place all these import fixes in another PR to reduce this one. I recommend to do the same in jet another  PR for the API doc files

Reply:
If I don't fix it here, the ESM build is broken. I discovered this issue while testing this PR. Since the change is minimal, hopefully it shouldn't make the review process much harder.

Reply:
Ok for the imports, but in general I would update API on separate PRs it makes the number of files too big. I find it easier to generate document updates PRs. Anyway we all know that API docs are not to be reviewed.